code/modules/reagents/chemistry/reagents/food.dm

```diff
@@ -782,6 +827,7 @@
 	description = "Scrapings of some unknown fungus found growing on the station walls."
 	reagent_state = LIQUID
 	color = "#C87D28"
+	taste_message = "puke"
```

Comment:
Moldy maybe? Pretty sure that's what we had before when you'd cook mushroom pizza before they removed fungus from one of the mushrooms. In fact wasn't that some kind of taste feature for mold alone?